src/DwitTech.AccountService.Core/Services/ActivationService.cs

```diff
@@ -0,0 +1,63 @@
+﻿using DwitTech.AccountService.Core.Interfaces;
+using DwitTech.AccountService.Core.Utilities;
+using Microsoft.AspNetCore.Http;
+using System;
+using System.Collections.Generic;
+using System.Linq;
+using System.Net;
+using System.Net.Mail;
+using System.Text;
+using System.Threading.Tasks;
+
+namespace DwitTech.AccountService.Core.Services
+{
+    public class ActivationService : IActivationService
+    {
+        private static string GetActivationCode()
+        {
+            var activationCode = RandomUtil.GenerateUniqueCode();
+            return activationCode;
+        }
+
+        private string GetNameFromDb(string userEmail) //TODO
+        {
+            return "Mike";
+        }
+
+        private static string GetBaseUrl() //TODO
```

Comment:
Who will implement this? The only pending task should be the SendMail. Okay, let the base url come from an environment variable. That should be more straightforward